sales_team_security/models/res_users.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © 2016 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    section_ids = fields.Many2many(
```

Comment:
Forward-looking, I'd call it `team_ids`.
https://github.com/odoo/odoo/commit/19551aec2ee3bf5e4c14c00058e6b41fea6dd429